tokio-rustls/tests/badssl.rs

```diff
@@ -71,7 +89,13 @@ async fn test_modern() -> io::Result<()> {
     let domain = "mozilla-modern.badssl.com";
 
     let (_, output) = get(config.clone(), domain, 443).await?;
-    assert!(output.contains("<title>mozilla-modern.badssl.com</title>"));
+    match output.contains("<title>mozilla-modern.badssl.com</title>") {
+        true => {}
+        false => {
+            eprintln!("{}", output);
+            panic!("failed badssl test")
+        }
+    }
```

Comment:
seems like this could just be an assert, still, but with the `output` moved into the assert message? like
```rust
assert!(output.contains("<title>mozilla-modern.badssl.com</title>"), "failed badssl test. output: {}", output);